tests/console/docker_image.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LINUX GmbH
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test installation and running of the docker image for this snapshot
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+use base 'consoletest';
+use testapi;
+use utils;
+use strict;
+
+sub run {
+    select_console 'root-console';
+
+    check_var('VERSION', 'Tumbleweed') || die 'Only Tumbleweed uses this test';
```

Comment:
ok but can we improve the die message to explain the reason? If I would see the message "Only Tumbleweed uses this test" then I would ask "so, what?" :)

Reply:
Well, that's literally the only reason. It would work for leap as well, but as the images aren't rolling it doesn't make sense.

The only technical reason is that the image and rpm names would differ and thus the test would fail. That's why the code looked different before, to make that more clear.

Reply:
My suggestion:

```perl
die 'At the moment only the tumbleweed docker image is tested';
```

I can imagine reusing this test for Leap 15 and SLE docker images

Reply:
That won't happen. Neither Leap nor SLE have docker images as part of snapshots.